backend/compact-connect/lambdas/license-data/tests/function/test_data_model/test_client.py

```diff
@@ -83,24 +162,26 @@ def test_get_licenses_sorted_by_family_name(self):
             jurisdiction='co'
         )
         self.assertEqual(100, len(resp['items']))
-        self.assertIn('lastKey', resp.keys())
+        self.assertIsInstance(resp['pagination']['lastKey'], str)
 
-        last_key = resp['lastKey']
+        last_key = resp['pagination']['lastKey']
         resp = client.get_licenses_sorted_by_family_name(  # pylint: disable=unexpected-keyword-arg,missing-kwoa
             compact='aslp',
             jurisdiction='co',
             pagination={'lastKey': last_key}
         )
-        self.assertEqual(100, len(resp['items']))
-        self.assertNotIn('lastKey', resp.keys())
+        # moto does not properly mimic dynamodb pagination in the case of an index with duplicate keys,
```

Comment:
I wasn't really thinking of this as a ToDo, so much as just noting why I'm not specifically verifying the ordering, because I have to work around a shortcoming in the underlying test/mock library. If they do wind up resolving the issue, we can of course refine the test here. I'm happy to handle it according to your preference.

Reply:
Got it. That makes sense. Good to leave as-is for now.